### PR TITLE
SSCS-9975: SSCS5 - Update Write Final Decision Event

### PR DESCRIPTION
**Pull request by srochani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/WriteFinalDecisionPreviewDecisionServiceBase.java [hunk 1]
```diff
@@ -150,6 +152,10 @@ protected NoticeIssuedTemplateBody createPayload(PreSubmitCallbackResponse<SscsC
 
     }
 
+    private void setUpOtherPartyAttendedQuestion(SscsCaseData caseData) {
+
```
Comment (divisathyan) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/WriteFinalDecisionPreviewDecisionServiceBase.java:156]: why do we need a void method?

**Review by divisathyan — COMMENTED** [review]
One comment

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/WriteFinalDecisionPreviewDecisionServiceBase.java (hunk 1 above)
Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/WriteFinalDecisionPreviewDecisionServiceBase.java]: Thanks removed it

**Review by divisathyan — APPROVED** [review]
LG

**Pull request merged**